packages/components/src/components/post-breadcrumbs/post-breadcrumbs.tsx

```diff
@@ -30,6 +30,12 @@ export class PostBreadcrumbs {
    */
   @Prop() menuLabel!: string;
 
+  /**
+   *
+   */
+
+  @Prop() readonly menuAnimation?: 'pop-in' | 'none';
```

Comment:
Should be a string or `undefined`.

```suggestion
  @Prop() readonly menuAnimation?: string;
```